### DISCUSSION TITLE
AWS JDBC Wrapper vs AWS RDS Proxy for DB failover handling

### DISCUSSION THREAD

**Discussion by ryszard-sch** [aws/aws-advanced-jdbc-wrapper#430] [Q&A] [answered]
I am currently seeking a solution for fast handling DB failover, and it seems that this can be accomplished by either setting up the AWS Advanced JDBC Wrapper or [configuring the AWS RDS Proxy](https://aws.amazon.com/blogs/database/improving-application-availability-with-amazon-rds-proxy/). I am using Postgres databases managed by AWS, including Aurora Postgres databases. Additionally, some of my applications utilize reactive database connectors.

Could you please advise me on which solution to pursue, and explain the reasons behind your recommendation?

**Comment by DaspawnW** [comment]
It seems like reactive support is missing. Are there any plans to add this support?

**Reply by davecramer** [reply, marked as answer]
> It seems like reactive support is missing. Are there any plans to add this support?

Currently this is not on the roadmap.  One challenge with reactive is which library to support ?
